sr_ethercat_hand_config/controls/motors/lh/motor_board_effort_controllers.yaml

```diff
@@ -1,340 +1,380 @@
 ffj0:
   backlash_compensation: true
   pid:
-    d: 64
+    d: 0
     deadband: 5
     f: 300
     i: 0
-    imax: 550
+    imax: 0
     max_pwm: 600
     p: 6200
     sg_left: 0
     sg_right: 0
     sgleftref: 0
     sgrightref: 0
     sign: 0
+    torque_limit: 500
+    torque_limiter_gain: 64
   pos_filter:
     tau: 0.05
 ffj3:
   backlash_compensation: true
   pid:
-    d: 64
+    d: 0
     deadband: 5
     f: 300
     i: 0
-    imax: 450
+    imax: 0
     max_pwm: 600
     p: 6200
     sg_left: 0
     sg_right: 0
     sgleftref: 0
     sgrightref: 0
     sign: 0
+    torque_limit: 450
+    torque_limiter_gain: 64
   pos_filter:
     tau: 0.05
 ffj4:
   backlash_compensation: true
   pid:
-    d: 64
+    d: 0
     deadband: 5
     f: 300
     i: 0
-    imax: 350
-    max_pwm: 300
+    imax: 0
+    max_pwm: 400
     p: 6200
     sg_left: 0
     sg_right: 0
     sgleftref: 0
     sgrightref: 0
     sign: 0
+    torque_limit: 350
+    torque_limiter_gain: 64
   pos_filter:
     tau: 0.05
 lfj0:
   backlash_compensation: true
   pid:
-    d: 64
+    d: 0
     deadband: 5
     f: 300
     i: 0
-    imax: 550
+    imax: 0
     max_pwm: 600
     p: 6200
     sg_left: 0
     sg_right: 0
     sgleftref: 0
     sgrightref: 0
     sign: 0
+    torque_limit: 500
```

Comment:
I copied the values from right->left, so this means the old left hand had a value different to the right. I suspect they should be the same. @jlaforga - can you answer this?